doc/man7/openssl-env.pod

```diff
@@ -51,6 +51,18 @@ See L<OPENSSL_malloc(3)>.
 Specifies the directory from which cryptographic providers are loaded.
 Equivalently, the generic B<-provider-path> command-line option may be used.
 
+=item B<OPENSSL_SEC_MEM>
+
+Initializes the secure memory at the beginning of the application which makes
+the secure memory calls not to fall back to regular memory calls. The value
+indicates the B<size> parameter in bytes. For further restrictions see
+L<CRYPTO_secure_malloc_init(3)>.
+
+=item B<OPENSSL_SEC_MEM_MINSIZE>
+
+An optional variable used with B<OPENSSL_SEC_MEM>. Default is 0. For more
```

Comment:
I think that this should spell out that this is passed as the _minsize_ argument like the previous paragraph does for _size_.